.travis.yml

```diff
@@ -1,4 +1,12 @@
 language: python
```

Comment:
This file, but also others in this PR are given executable mode. Is it needed?

Reply:
My bad, I'll fix it.